### PR TITLE
Formatting fixed

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -60,49 +60,46 @@ lazy val csv = alpakkaProject("csv", Dependencies.Csv)
 
 lazy val dynamodb = alpakkaProject("dynamodb", Dependencies.DynamoDB)
 
-lazy val elasticsearch = alpakkaProject("elasticsearch", Dependencies.Elasticsearch,
+lazy val elasticsearch = alpakkaProject(
+  "elasticsearch",
+  Dependencies.Elasticsearch,
```
Comment (raboof) [inline build.sbt]: Hm the existing manual formatting seemed more consistent to me.. but oh well ;)

**Comment by johanandren** [comment]
What, now one of the builds succeeded and then another one failed with formatting O.o

**Comment by johanandren** [comment]
Nah, that is always in the output perhaps. it's the cassandra spec failing

**Comment by johanandren** [comment]
Merging this since the repeating failure is the cassandra spec instability tracked by #527

**Pull request merged**